src/styles/vars.less

```diff
@@ -1,6 +0,0 @@
-@import '~antd/lib/style/themes/default.less';
-@primary-color: #1b8ad3;
-@border-radius-base: 10px;
-@font-family: QuickSand, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
-    'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji',
-    'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
```

Comment:
I'm also assuming these 3 variables need to be moved to `default-vars.less`, right?

I'll pull these in after I merge the PR.

Reply:
To my astonishment, these variables were defined but never applied - as I have found out in the old implementation.

Reply:
That's really bizarre. I noticed they are gone after I merged because the rounded corners were gone. 

But after I added [these variables back](https://github.com/caprover/caprover-frontend/commit/d06103e63a96471f6da149ae37c83e984bf5f2a4), the new rounded corners came back. You can see the new design here: https://captain.server.demo.caprover.com

Reply:
There was clearly a misunderstanding here from my side. I have not seen the rounded borders introduced since the last release because I compared them with the last release on my prod and noticed on the dev that the boxes are too round.